_sources/Unit1-Using-Objects-and-Methods/topic-1-12-objects.rst

```diff
@@ -369,21 +376,157 @@ two turtle objects are created: ``yertle`` and ``myrtle``.  You can name your tu
         }
     }
 
-Here is a class diagram that shows some of the attributes and methods in the class ``Turtle``.
 
-.. creately.com for figure
+Class Hierarchy and Inheritance
+--------------------------------
+
+.. index::
+    single: inheritance
+    single: superclass
+    single: subclass
+    single: Object
+
+Another important concept in object-oriented programming is **inheritance**. Although the AP CSA exam no longer covers inheritance, you should know what it means. **Inheritance** is a way to create a new class that is based on an existing class. The new class, called a **subclass**, inherits the attributes and behaviors of the existing class, called a **superclass**.  In Java, all classes are subclasses of a superclass called **Object**. 
```

Comment:
While I approve of teaching more than the College Board wants to, I wonder if we need this? If we never come back to talk more about inheritance I wonder if it's worth giving them just this little bit which is mostly going to raise questions that won't be answered later?

Reply:
I was just about to write to you about this. Unfortunately it's an EK. I did not want to add it!
EK 1.12.A.2	A class hierarchy can be developed by putting common attributes and behaviors of related classes into a single class called a superclass. Classes that extend a superclass, called subclasses, can draw upon the existing attributes and behaviors of the superclass without replacing these in the code. This creates an inheritance relationship from the subclasses to the superclass. X EXCLUSION STATEMENT—Designing and implementing inheritance relationships are outside the scope of the AP Computer Science A course and exam.)

Reply:
Oh. I guess they had to leave at least that much in otherwise how do you even begin to describe equals and toString.